invenio_records_rest/__init__.py

```diff
@@ -22,7 +22,272 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-"""REST API for Records."""
+u"""REST API for Records.
+
+Invenio-Records-REST is a core component of Invenio which provides configurable
+REST APIs for searching, retrieving, creating, modifying and deleting records.
+
+The module uses Elasticsearch as the backend search engine and builds on top
+a REST API that supports features such as:
+
+- Search with sorting, filtering, aggregations and pagination which allows the
+  full capabilities of Elasticsearch to be used such as e.g. geo-based quering.
+- Record serialization and transformation (e.g. JSON, XML, DataCite,
+  DublinCore, JSON-LD, MARCXML, Citation Style Language).
+- Pluggable query parser.
+- Tombstones and record redirection.
+- Customizable access control.
+- Configurable record namespaces for exposing different classes of records
+  (e.g.authors, publications, grants, ...).
+- CRUD operations for records with support for persistent identifier minting.
+- Super-fast completion suggesters for implementing Google like instant
+  autocomplete suggestions.
+
+The REST API follows best practices and supports e.g.:
+
+- Content negotiation and links headers.
+- Cache control via ETags and last modified headers.
+- Optimistic concurrency control via ETags.
+- Rate-limiting, Cross-Origin Resource Sharing, and various security headers.
+
+The Search REST API works as **the** central entry point in Invenio for
+accessing records. The REST API in combination which e.g. Invenio-Search-UI/JS
+allows to easily display records anywhere in Invenio and still only maintain
+one single search endpoint.
+
+For further extending the REST API take a look at the guide in Invenio-REST:
```

Comment:
``extending the`` -> ``extending of the``

Reply:
This is not correct.